server/GlideElement.d.ts

```diff
@@ -1,3 +1,144 @@
-import { SNAPIGlideElement } from "./SNAPIGlideElement";
-declare class GlideElement extends SNAPIGlideElement {}
+import { SNAPIGlideElement } from './SNAPIGlideElement';
+declare class GlideElement extends SNAPIGlideElement {
+  // Methods to access currency fields
+  // TODO:
+  //      This should be moved in separate class, but this requires work in other packages.
+  //      See for more info: https://github.com/nuvolo/servicenow-types/issues/22
+
+  /**
+   * For Currency fields only!
+   *
+   * Returns the currency value in the user's session currency as an unformatted number.
+   *
+   * Example: "1563.72"
+   */
+  getValue(): string;
+
+  /**
+   * For Currency fields only!
+   *
+   * Returns the currency value in the reference currency as an unformatted number.
+   *
+   * Example: "1152.48"
+   */
+  getReferenceValue(): string;
+
+  /**
+   * For Currency fields only!
+   *
+   * Returns the currency value in the user's session currency as and unformatted number.
+   *
+   * Example: "1563.72"
+   */
+  getSessionValue(): string;
+
+  /**
+   * For Currency fields only!
+   *
+   * Returns the currency value as entered as an unformatted number.
+   * Note: This might not be the session or reference currency.
+   *
+   * Example: "21345.67"
+   */
+  getCurrencyValue(): string;
+
+  /**
+   * For Currency fields only!
+   *
+   * Returns the currency value in the user's session currency
+   * formatted in the user's locale with a currency symbol.
+   *
+   * Example: "€1.563,72"
+   */
+  getDisplayValue(): string;
```

Comment:
getDisplayValue exists on the base GlideElement so let's keep it out of the override for now. I think there are a few other methods like this one in this commit, can you go through real quick and limit the additional methods to those that don't currently exist in the GlideElement definition?